pages/linux/sic.md

```diff
@@ -0,0 +1,25 @@
+# sic
+
+> Simple IRC client.
+> Part of the suckless tools.
+> More information: <https://tools.suckless.org/sic/>.
+
+- Connect to a host with the specified nickname:
+
+`sic -h {{irc.ofct.net}} -n {{$USER}}`
+
+- Connect to a different host with PASS connection password:
```

Comment:
I'm confused — what does the uppercase `PASS` refer to here?

Reply:
tbh, it's a copy paste from suckless docs; don't mind to remove the PASS part, turn it into irc password or something?

Reply:
If my previous edit suggestion reaches consensus, then we could use matching language here as well, for consistency:

```suggestion
- Connect to a given host, using a given nickname and password:
```

Though before we make this change, I think we need to clarify what happens in the previous example (passing `-n` but not `-p`) — does sic expect the password to be in an environment variable too (say, `$PASS`)? Does it ask for it interactively?

Reply:
I don't use it like that normally, but just tested this out with `$PASS` set as an environment variable without `-k {{pass}}`. Does not work. So I'll commit your suggestion.